kinetic/distribution.yaml

```diff
@@ -6529,6 +6529,17 @@ repositories:
       url: https://github.com/allenh1/p2os.git
       version: master
     status: maintained
+  pacifica_dbw_ros:
+    doc:
+      type: git
+      url: https://github.com/NewEagleRaptor/pacifica-dbw-ros.git
+      version: default
```

Comment:
This needs to be a valid branch name, In this case: `master`

Reply:
Thank you.  I realized that at the same time I saw you post the comment.